demo/src/lib/gtag.js

```diff
@@ -0,0 +1,24 @@
+
+// eslint-disable-next-line no-undef
+const GA_TRACKING_ID = process.env.REACT_APP_GA_TRACKING_ID;
+
+/**
+ * Sends a page_view event to Google Analytics.
+ * @param {string} url
+ */
+export const pageview = (url) => {
+  window.gtag('config', GA_TRACKING_ID, {
+    page_path: url,
+  });
+};
+
+/**
+ * Sends an event to Google Analytics.
```

Comment:
A small note.
It will send an event to all of the targets previously configured using "config" command (unless parameters specify the target explicitly using "send_to" property).
For example if there's a gtag('config', 'AW-123') called before, it will also send data to Google Ads. If no "config" has been called then nothing will be sent.
So the statement might not always be true and depends on previous invocations as well as parameters value.

You should consider rephrasing or adding `parameters['send_to'] = GA_TRACKING_ID` in the method to guarantee that data will be sent to your GA property.

Reply:
Thank you, I didn't realize that is how it worked to determine where to send events to. I added the explicit parameter.